eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -23,6 +23,25 @@ steps:
       # Define this inline, because of the chicken/egg problem with loading a script when nothing
       # has been checked out yet.
       script: |
+        function Clone([Hashtable]$repository)
```

Comment:
Should we also have some variable that we can detect that will override and force us to use Clone instead of even trying the sparse checkout?

Reply:
Are you thinking this would be a pipeline variable, e.g. set in globals or added to the UI by the generator?

Reply:
Yeah some sort of variable that we could set to turn off sparse checkout, either at queue time or globally in the repo via globals.yml or some sort.

Reply:
Yeah some sort of variable that we could set to turn off sparse checkout, either at queue time or globally in the repo via globals.yml or some sort.